shared/common-adapters/squeeze.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react'
+import {Box2} from './box'
+import * as Styles from '../styles'
+
+const Squeeze = (props: {children?: React.ReactNode; enable: boolean}) =>
+  props.enable ? (
```

Comment:
do you ever see this being used with something other than `isTablet` as `enable`? if not, then you should delete the prop and just hard code that.